data/Store.js

```diff
@@ -97,11 +97,11 @@ export class Store {
 
     /**
      * Remove a record (and all its children, if any) from the store.
-     * @param {(string|number)} id - ID of the the record to be removed.
+     * @param {(string[]|number[])} ids - IDs of the records to be removed.
      */
     @action
-    removeRecord(id) {
-        this._all = this._all.removeRecord(id);
+    removeRecords(ids) {
```

Comment:
We could do a `ids = castArray(ids)'` here just to make it nice and clean for apps to remove a single record (something I tend to do a lot in inline editable grids)